.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+__Issues are used only for bug reports and feature requests.__
+
+If you have troubles with JanusGraph installation, configuration and integrations or you have questions about JanusGraph usage use this group to ask your questions:  
```

Comment:
I think there should be comma after _usage_ as it's the end of the if clause. Maybe it would also sound a bit better if the start would be like this:
_If you have trouble with the installation or configuration of JanusGraph or [...]_
?